### ISSUE TITLE
Build in further accessibility checks

### ISSUE THREAD

**Issue by ashhath01** [deptofdefense/dds.mil#135]

**Comment by jhackshaw-dds** [comment]
One off lighthouse reports and A11Y checks look great. If we automate these, I think it should be done outside of the netlify build step so as not to slow down deploys.

![image](https://user-images.githubusercontent.com/59582489/86927957-f2272080-c101-11ea-86aa-e573b92a8323.png)
